styles.css

```diff
@@ -3,78 +3,103 @@
     padding: 0;
     box-sizing: border-box;
 }
-.app {
-    width: 100%;
-    height: 100%;
-    background: url(images/background.png);
-    background-repeat: no-repeat;
-    background-size: cover;
-}
+
 .vector {
     list-style: none;
     display: flex;
     gap: 10px;
 }
-.nav-bar-list {
-    display: flex;
-    gap: 32px;
-   align-items: center;
-}
-.nav-bar-list-right {
+.feature-section {
     display: flex;
-    gap: 12px;
-   align-items: center;
+    flex-direction: column;
+    padding: 64px 0 0;
+    background: #FFFFFF;    
 }
-.nav-bar {
-    height: 72px;
+.feature-section-full {
     display: flex;
+    flex-direction: column;
+    gap: 50px;
     align-items: center;
-    justify-content: space-around;
-    background: #FFFFFF;
 }
-.nav-link {
-    list-style: none;
+.feature-header {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    box-shadow: 0px 4px 4px rgb(0 0 0 / 25%);
 }
-
-.nav-link-text {
+.feature-head-main {
     font-family: 'Inter', sans-serif;
-    font-weight: 500;
-    font-size: 14px;
-    line-height: 20px;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 64px;
+    text-align: center;
     color: #18191F;
-    text-decoration: none;
-}
-.hero-section {
-    margin: 100px auto auto;
+    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
 }
-.hero-title {
+.feature-head-sub {
+    width: 540px;
     font-family: 'Inter', sans-serif;
-    font-weight: 800;
-    font-size: 72px;
-    line-height: 98px;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 32px;
+    text-align: center;
     color: #18191F;
-    margin-top: 70px;
-    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);    
 }
-.hero-content {
-    gap: 32px;
+.features-list {
+    display: flex;
+    flex-direction: column;
+    gap: 64px;
+}
+.features-list-firstrow {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 30px;   
+}
+
+.features-list-icon {
+    width: 32px;
+    height: 32px;
+}
+.features-list-text {
     display: flex;
     flex-direction: column;
+    align-items: center;
+    padding: 0px;
+    gap: 8px;
+    width: 350px;
+    height: 92px;
 }
-.hero-button {
-    width: 154px;
-    height: 48px;
-    background: #8C30F5;
-    border-radius: 6px;
-    color: #FFFFFF;
-    border: 0;
+.features-list-main {
+    width: 350px;
+    height: 32px;
     font-family: 'Inter', sans-serif;
-    font-weight: 500;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+}
+.features-list-sub {
+    width: 350px;
+    height: 52px;
+    font-family: 'Inter', sans-serif;
```

Comment:
you can define font-family in top most parent container so that it will be applied to all child container's text-content instead of applying for text content classes.